var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -91,6 +91,7 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
         multi=False,
         description="global ordinal type for Tpetra",
     )
+    variant("ninja", default=False, description="Enable Ninja makefile generator")
```

Comment:
Similiar variants have been proposed and rejected for several projects. Since it changes only the build system and shouldn't change the installed binaries, it shouldn't be a variant. Since ninja is better than make in most cases it might be good to have a global spack option to default the cmake generator to something different: I'll discuss this with the maintainers.

Reply:
Makes sense. I'll just remove this. Was curious how someone else felt about it here.